guides/release/testing/different-types-of-tests.md

```diff
@@ -0,0 +1,200 @@
+In Ember, there are 3 types of tests. From the least complex to the most, we have:
+
+- Unit tests
+- Rendering tests (integration tests)
+- Application tests (acceptance tests)
+
+Broadly speaking, these tests differ in two aspects. (1) How much of our app should they cover for correctness? In other words, we look at separation of concerns. (2) How fast can they run? After all, we want our tests to finish in a timely manner.
```

Comment:
Though I agree with and like your technical explanation, here, in an introduction, I think it's better to present a layman one. I changed the paragraph so that I could incorporate parts of your suggested paragraph, thanks!